dc-cudami-editor/src/components/lists/PagedIdentifiableList.jsx

```diff
@@ -69,6 +70,7 @@ class PagedIdentifiableList extends Component {
       identifierTypes,
       numberOfPages: Math.ceil(totalElements / pageSize),
       totalElements,
+      showSearch: totalElements > 0,
```

Comment:
If the list is empty from the very beginning on we wanna hide the search field.

Reply:
Guter Punkt :+1: